### ISSUE TITLE
agama-cli fails in `%check` on aarch64

### ISSUE THREAD

**Issue by ggardet** [openSUSE/agama#828]
agama-cli fails in `%check` on aarch64:
```
[  430s] running 3 tests
[  431s] test test_update_connection ... FAILED
[  431s] test test_read_connections ... ok
[  431s] test test_add_connection ... ok
[  431s] 
[  431s] failures:
[  431s] 
[  431s] ---- test_update_connection stdout ----
[  431s] thread 'test_update_connection' panicked at agama-dbus-server/tests/network.rs:118:61:
[  431s] called `Result::unwrap()` on an `Err` value: DBus(MethodError(OwnedErrorName(ErrorName(Str(Owned("org.freedesktop.DBus.Error.UnknownInterface")))), Some("Unknown interface 'org.opensuse.Agama1.Network.Connections'"), Msg { type: Error, sender: UniqueName(Str(Borrowed(":1.0"))), reply-serial: 9, body: Signature("s") }))
[  431s] note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[  431s] 
[  431s] 
[  431s] failures:
[  431s]     test_update_connection
[  431s] 
[  431s] test result: FAILED. 2 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.85s
[  431s] 
[  431s] error: test failed, to rerun pass `-p agama-dbus-server --test network`
[  431s]      Running unittests src/lib.rs (target/debug/deps/agama_derive-20f7a9774951ff3a)
[  431s] 
[  431s] running 0 tests
[  431s] 
[  431s] test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.01s
[  431s] 
[  431s]      Running unittests src/lib.rs (target/debug/deps/agama_lib-02116d59f1447683)
```
Full log at: https://build.opensuse.org/package/live_build_log/systemsmanagement:Agama:Devel/agama-cli/openSUSE_Tumbleweed/aarch64

It seems the problem started with https://build.opensuse.org/package/rdiff/systemsmanagement:Agama:Devel/agama-cli?orev=2d672b39ae6d9044a7386e2ce55550af&rev=fa82b46a2f6b41867b0a8c64869bb430 which is basically https://github.com/openSUSE/agama/pull/812

**Comment by imobachgs** [comment]
Hi @ggardet,

Thanks for reporting! While investigating the issue, I saw an additional problem with `s390x` too (although slightly different). #834 should fix the `s390x` problem, but not the one with `aarch64`. For some reason, the tests are timing out. However, reducing the number of parallel tests to just 1 does the trick. I have been able to build the package in `aarch64` a couple of times.

Perhaps changing the number of jobs for `aarch64` is a plausible workaround by now.
